src/components/NoteExpand/CommentsBox/CommentsList/CommentsList.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import Comment from './Comment/Comment';
+import { CommentsListProps } from './CommentsList.types';
+
+const CommentsList: React.FC<CommentsListProps> = ({ comments }) => {
+  return (
+    <>
+      {comments.map((comment) => (
+        <Comment key={comment.id} author={comment.author} content={comment.content} created_at={comment.created_at} />
```

Comment:
no need to pass every comment field separately just pass `comment={comment}`

Reply:
ready